src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMaxTable.cs

```diff
@@ -23,5 +28,20 @@ public MinMaxTableFunction(bool isMin)
         {
             _delegationCapability = isMin ? DelegationCapability.Min : DelegationCapability.Max;
         }
+
+        public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+
+            var fValid = CheckInvocation(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+
+            // The return type will be the type of whichever field was chosen to min/max on.
```

Comment:
`will be the result type of expression in the 2nd argument`, since it is a lambda, not necessarily a field name